flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/source/FlinkSource.java

```diff
@@ -107,6 +114,11 @@ public Builder limit(long newLimit) {
       return this;
     }
 
+    public Builder locality(boolean locality) {
```

Comment:
similarly we can improve the naming once we settled on the config name.

Reply:
how about 'exposeLocality'?